includes/Core/REST_API/Exception/Invalid_Dimensions_Exception.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\REST_API\Exception\Invalid_Dimensions_Exception
+ *
+ * @package   Google\Site_Kit\Core\REST_API\Exception
+ * @copyright 2022 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\REST_API\Exception;
```

Comment:
Let's move this to (a new) `Core\Validation\Exception` namespace as `REST_API` is rather specific to that integration, although this validation is applicable more generally (e.g. if SK API requests were possible via WP-CLI).

TBH this could apply to the datapoint exception too, but let's leave that be for now 😄